scripts/parse.js

```diff
@@ -29,18 +29,22 @@ fs.readdir(CHAINS_DIRECTORY, async function (err, files) {
       if (fileStat.isFile() && ext === '.json') {
         let json = require(filePath)
 
-        let newCoins = json.coins.map(coin => ({
+        let newCoinsList = json.coins.list.map(coin => ({
           name: coin.name,
           symbol: coin.symbol,
           denom: coin.denom,
-          exponent: coin.exponent
+          exponent: coin.exponent,
+          types: coin.types
         }))
 
         let newJson = {
           name: json.name,
           networkId: json.networkId,
           interface: json.interface,
-          coins: newCoins,
+          coins: {
+            defaultTypes: json.coins.defaultTypes,
```

Comment:
What's the pupuse of this field? Can't we simplify it to `coins` being an array directly?

Reply:
I think I found the answer in the ticket.

But a flag that is set for all coins does not need to be set, right? We either need it (like bondable) or it is implied by "coins" anyways (like sendable, fungible). I'd rather add some more flags to all coints in the list than making the structure more complex.

Reply:
I personally think that having a single `defaultTypes` field on the top would result in less repetition later. Also, having the user to duplicate the flags inside each coin could result in data be lost. 

BTW, I don't have a strong opinion on this so we could change it easily. Would like to know what @pedrouid thinks about it tho.

Reply:
Let's keep the structure simple for now. When a more complex structure becomes necessary, we can design it. At that point we understand the requirements far better than now